project.clj

```diff
@@ -92,6 +104,7 @@
                          [puppetlabs/jdbc-util "1.0.0"]
                          [puppetlabs/typesafe-config "0.1.5"]
                          [puppetlabs/ssl-utils "0.8.3"]
+                         [puppetlabs/jruby-utils "0.8.0"]
```

Comment:
Yeah, looking at the current HEAD of pe-aio, I think if you change its current puppetserver dependency from 2.7.2 to 5.0.0-master-20170329_182047-ga16e296 that the pedantic warning would go away.